src/core/chunked_stream.js

```diff
@@ -267,24 +242,483 @@ var ChunkedStream = (function ChunkedStreamClosure() {
         return missingChunks;
       };
       var subStream = new ChunkedStreamSubstream();
+      subStream.ensureRange(start, start + length);
```

Comment:
Does this mean we will remove incremental parsing optimization made by #3164. See benefits https://air.mozilla.org/intern-pdfjs/